Assets/Scripts/UI/InGame/Graphics/GraphicSettingMenu.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Assets.Scripts.UI.InGame
+{
+	public class GraphicSettingMenu : MonoBehaviour {
+
+		public GraphicsController GraphicController;
+		public InGameMenu Menu;
+		public GameObject QualityMenu;
+		public GameObject ResolutionMenu;
+
+		private void OnDisable() {
+			GraphicController.label.text = "";
+		}
+
+		public void ShowQualityMenu()
+		{
+			if(QualityMenu.GetActive() == true)
```

Comment:
`if(Qualitymenu.GetActive()` does the same as `if(QualityMenu.GetActive() == true)`